docker-compose.yml

```diff
@@ -21,11 +23,15 @@ services:
       MYSQL_ALLOW_EMPTY_PASSWORD: 1
     ports: 
       - "3306:3306"
+    volumes:
+      - nodemongodata:/data/db
 
   redis:
     image: redis
     ports:
       - "6379:6379"
+    volumes:
```

Comment:
I don't think we need this for redis, IIRC persistence is disabled by default in the docker redis image